rpcserver.go

```diff
@@ -2446,6 +2446,13 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 	}
 
 	amt := btcutil.Amount(invoice.Value)
+	
+	// The value of the invoice can not be zero or negative
+	if amt <= 0 {
```

Comment:
Invoices _can_ be zero, but they of course shouldn't be negative. A zero valued invoice just means that the payer can specify the amount, so think of the donation use case.

Reply:
cool, fixed.